blazor/treeview/full-row-navigable.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Navigation in Blazor TreeView Component | Syncfusion
+description: Checkout and learn here all about FullRowNavigable in Syncfusion Blazor TreeView component and much more.
+platform: Blazor
+control: TreeView
+documentation: ug
+---
+
+# Navigation in Blazor TreeView Component
+
+In the TreeView component, we have provided a **NavigateUrl** property. By using this property, you can navigate to other pages with TreeView node selection.
+
+The TreeView `FullRowNavigable` property is used to make the entire TreeView node navigable instead of text element in the Blazor TreeView component.

```

Comment:
Map the API

Reply:
The NavigateUrl property is missing from the blazor documentation API link. 

Reply:
Done.